product_docs/docs/bdr/4/camo.mdx

```diff
@@ -1,108 +1,102 @@
 ---
-navTitle: Commit at Most Once
-title: Commit At Most Once
+navTitle: Commit at most once
+title: Commit at most once
```

Comment:
I don't agree with lower-case here given the feature is literally named "Commit at Most Once (CAMO)" and not "commit at most once"

Reply:
I.e. it's not description, it's a name of an algorithm, just like Eager Replication or Group Commit